client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/eligibility-warnings-modal.tsx

```diff
@@ -0,0 +1,37 @@
+import { Dialog } from '@automattic/components';
+import EligibilityWarnings from 'calypso/blocks/eligibility-warnings';
+import type { SiteDetails } from '@automattic/data-stores';
+
+export const EligibilityWarningsModal = ( {
+	site,
+	isMarketplace,
+	isOpen,
+	handleClose,
+	handleContinue,
+}: {
+	site?: SiteDetails;
+	isMarketplace?: boolean;
+	isOpen: boolean;
+	handleClose: () => void;
+	handleContinue: () => void;
+} ) => {
+	return (
+		<Dialog
+			additionalClassNames="plugin-details-cta__dialog-content"
+			additionalOverlayClassNames="plugin-details-cta__modal-overlay"
```

Comment:
It might be a good idea to change those class names.

Reply:
Good point. I have extracted those to a specific `scss` file that accompanies the main `tsx` file so the specific styles of the Dialog can be defined there. Commit https://github.com/Automattic/wp-calypso/pull/81412/commits/065c0cef06abca8c2ac6bca4c7cf302bc5719855

Reply:
Great, ty.